guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p>Some changes to <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html#distance-allocation" target="_blank">distance_allocation </a>as well. </p><ul><li>Adds parameters to&nbsp;<a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html#distance-allocation" target="_blank">distance_allocation</a></li><li class="ql-indent-1"><code>output_distance_accumulation_raster_name</code></li><li class="ql-indent-1"><code>output_back_direction_raster</code></li><li class="ql-indent-1"><code>output_source_direction_raster_name</code></li><li class="ql-indent-1"><code>output_source_location_raster_name</code></li><li>Removes parameter from&nbsp;<a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html#distance-allocation" target="_blank">distance_allocation</a></li><li class="ql-indent-1"><code>generate_source_row_column_bands</code></li><li>Enhances&nbsp;<a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html#distance-allocation" target="_blank">distance_allocation&nbsp;</a>to generate multiple outputs</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Edited the notes to include this info.</p>